src/devices/QwiicButton/QwiicButton.QueueManipulation.cs

```diff
@@ -0,0 +1,119 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+using Iot.Device.QwiicButton.RegisterMapping;
+
+namespace Iot.Device.QwiicButton
+{
+    public sealed partial class QwiicButton
+    {
+        /// <summary>
+        /// Returns whether the queue of button press timestamps is full.
```

Comment:
does the register actually have a timestamp? is it public? if not then do not mention timestamp in the doc

Reply:
Queue contains timestamps that you can pop.